iphone/Classes/TiUITextField.m

```diff
@@ -305,6 +305,17 @@ - (void) dealloc
 		NSNotificationCenter * theNC = [NSNotificationCenter defaultCenter];
 		[theNC addObserver:self selector:@selector(textFieldDidChange:) name:UITextFieldTextDidChangeNotification object:textWidgetView];
 	}
+  
+    // TIMOB-16100: Native issue that prevents the textfield to mutate the font-config
+    BOOL needsAdjustment = (![TiUtils isIOS10OrGreater] && ((UITextField *)textWidgetView).secureTextEntry);
+    
+    if (needsAdjustment)
+    {
+        NSString *str = ((UITextField *)textWidgetView).text;
+        ((UITextField *)textWidgetView).text = @"";
+        ((UITextField *)textWidgetView).text = str;
+    }
```

Comment:
Please adjust the indentation to the one used in this source (it looks like the legacy tabs have been used). The rule it that we use tabs for old code that uses it and spaces for new code that already uses that. Thx!